spec/spec_helper.rb

```diff
@@ -8,15 +8,16 @@
 require 'webmock/rspec'
 require 'rspec/its'
 
+require 'pry'
```

Comment:
Is this left here intentionally? :)

Reply:
No, it must be detritus from a previous commit

Reply:
FYI, I certainly do not mind it if people commit their fixes directly to my PR branch.

Reply:
This was left here intentionally, so that `binding.pry` would actually work.

This development dependency was added in 2013 via https://github.com/pivotal/LicenseFinder/commit/2af198f67fc5ffc2c9fa26809528b8c92ef0197c, and while no context was captured as to why it was added, adding this line enables interactive debugging which I find most useful: https://github.com/pry/pry#runtime-invocation

You may either want to:
* remove pry altogether - this was done before in https://github.com/pivotal/LicenseFinder/commit/d17495d863ec11a48ef22889a3e108f3e24b0fb9, but then reintroduced in https://github.com/pivotal/LicenseFinder/commit/2af198f67fc5ffc2c9fa26809528b8c92ef0197c
* add the `require 'pry'` line back so that using pry in development (great help when TDD-ing) is just a `binding.pry` away

Reply:
Thanks for the explanation @gerhard - 448ea897fe1ffad55879d285323b2cb372b82af1